MassSpectrometry/MsDataFile.cs

```diff
@@ -101,7 +101,7 @@ public static void WindowModeHelper(ref double[] intensities, ref double[] mArra
             //windowNum is the number of windows
```

Comment:
maybe skip block copy if(filteringParams.NumberOfWindows == 1) ?

Reply:
Yea, I'll do it right away when I'm back around 4. Thanks!